rel/config.exs

```diff
@@ -0,0 +1,56 @@
+# Import all plugins from `rel/plugins`
+# They can then be used by adding `plugin MyPlugin` to
+# either an environment, or release definition, where
+# `MyPlugin` is the name of the plugin module.
+~w(rel plugins *.exs)
+|> Path.join()
+|> Path.wildcard()
+|> Enum.map(&Code.eval_file(&1))
+
+use Mix.Releases.Config,
+  # This sets the default release built by `mix release`
+  default_release: :default,
+  # This sets the default environment used by `mix release`
+  default_environment: Mix.env()
+
+# For a full list of config options for both releases
+# and environments, visit https://hexdocs.pm/distillery/config/distillery.html
+
+# You may define one or more environments in this file,
+# an environment's settings will override those of a release
+# when building in that environment, this combination of release
+# and environment configuration is called a profile
+
+environment :dev do
+  # If you are running Phoenix, you should make sure that
+  # server: true is set and the code reloader is disabled,
+  # even in dev mode.
+  # It is recommended that you build with MIX_ENV=prod and pass
+  # the --env flag to Distillery explicitly if you want to use
+  # dev mode.
+  set(dev_mode: true)
+  set(include_erts: false)
+  set(cookie: :"<pz:GPgNHnx}YDmV@<2ce]Vp`lxN~mvZew<R0,w8>*Q~dxLI=,G)?(^E{R[^_Scw")
+end
+
+environment :prod do
+  set(include_erts: true)
+  set(include_src: false)
+  set(cookie: :"$<v9JtiT1<RK},ayQRyUFLDfxFxh$(_%6stPoo)Fk%>h[tV!M]u~Aj1V|;%$[=K4")
```

Comment:
Note: this is actually "OK" to commit, Gigalixir will set a secure cookie + vm.args file for us. We would have to manually turn it off to override the default Gigalixir configuration (which you can totally do if you want, just don't commit the cookie or randomly generate or something).